package-lock.json

```diff
@@ -100645,9 +100645,9 @@
       }
     },
     "ws": {
-      "version": "5.2.3",
-      "resolved": "https://registry.npmjs.org/ws/-/ws-5.2.3.tgz",
-      "integrity": "sha512-jZArVERrMsKUatIdnLzqvcfydI85dvd/Fp1u/VOpfdDWQ4c9qWXe+VIeAbQ5FrDwciAkr+lzofXLz3Kuf26AOA==",
+      "version": "5.2.2",
```

Comment:
So there is an issue with that, I've merged some Maintenance changes that add 5.2.3 version to the lock file, but if you run npm install with npm version 8.0.0 which is how it's supposed to be, this version gets downgraded. So I guess this issue can be tackled separately. I'm not sure the npm version in our image on git is correct since it leaves it at 5.2.2. 

So I did leave this on purpose so that I don't have to exclude the file from commit every time until this is fixed. But I can revert the change if you feel strongly about it